service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGenerator.java

```diff
@@ -28,7 +29,7 @@
 
         Map<String, Object> data = new HashMap<>();
 
-        if (order.isOrderFinal()) {
+        if (IsFinalOrder.YES.equals(order.getIsFinalOrder())) {
```

Comment:
the logic should be if it's final order or is maybe and selected as final order, like the other implmenented in the close class